.storybook/grid/GridPanel.tsx

```diff
@@ -34,16 +51,36 @@ export const GridPanel = ({ api, active }: Props) => {
             return;
         }
 
-        const cropAssets = data.crop.data.assets;
-        const cropUrl = cropAssets[cropAssets.length - 1]?.secureUrl;
+        const cropAsset = getBestCrop(data.crop.data.assets.concat([]));
```

Comment:
What does the `concat` get us here?

Reply:
I think before the array is a weird read only type which for some reason doesn't have the sort method on it.

Reply:
Ah cool thanks for explaining 👍  I wonder if we should add a comment saying the same thing.

Reply:
I've added a comment explaining this. I also updated `getBestCrop` to use the `Asset` type from `@guardian/grid-client` instead of (re)defining it ourselves.